source/Patches/EndGameSummary.cs

```diff
@@ -33,14 +39,49 @@ internal class PlayerInfo
             }
         }
 
+        [HarmonyPatch(typeof(HudManager), nameof(HudManager.Update))]
+        [HarmonyPriority(Priority.Last)]
+        public static void Postfix(HudManager __instance)
+        {
+            if (EndGameSummary.AdditionalTempData.updateFlag)
```

Comment:
Instead of using the boolean flag, can we get this from the game status somehow?

Reply:
i will have a look at this when i get the chance as im not a big fan of using a "flag" variable either.

Reply:
also.. the "start" RPC is where i put the flag change... so im kind of using the change of game state already.